src/views/Dashboard.vue

```diff
@@ -243,20 +246,26 @@ export default {
       return sessionStorage.getItem('EDIT_URL')
     },
 
-    /** Show Missing Information alert only for firm with missing info warning. */
+    /** Whether to show Missing Information alert. */
     isMissingInformationAlert (): boolean {
-      return this.isFirm && this.hasMissingInfoWarning
+      return this.hasMissingInfoWarning
     },
 
-    /** Show Not In Good Standing alert only for firm with compliance warning. */
+    /** Whether to show Not In Compliance alert. */
+    isNotInComplianceAlert (): boolean {
+      return this.hasComplianceWarning
+    },
+
+    /** Whether to show Not In Good Standing alert. */
     isNotInGoodStandingAlert (): boolean {
-      return this.isFirm && this.hasComplianceWarning
```

Comment:
This is the wrong logic or the wrong alert, so I fixed it (re: the new compliance alert component).